types/event-source-polyfill/index.d.ts

```diff
@@ -0,0 +1,82 @@
+// Type definitions for event-source-polyfill 1.0
+// Project: https://github.com/Yaffle/EventSource
+// Definitions by: Emily Klassen <https://github.com/forivall>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.4
+
+declare global {
+    // Declare empty stub interfaces for environments where "dom" lib is not included
+    interface EventSource {}
+    interface EventSourceInit {}
+}
+
+export interface EventSourcePolyfillInit {
+    withCredentials?: boolean;
+    lastEventIdQueryParameterName?: string;
+    heartbeatTimeout?: number;
+    headers?: { [name: string]: string };
+    Transport?: new () => any;
+}
+
+export interface Event {
+    type: string;
+    target: any;
+}
+
+export interface EventListener {
+    (evt: Event): void;
+}
+
+export interface EventListenerObject {
+    handleEvent(evt: Event): void;
+}
+
+export type EventListenerOrEventListenerObject = EventListener | EventListenerObject;
+
+export interface EventSourceEventMap {
+    "error": Event;
+    "message": MessageEvent;
+    "open": Event;
+}
+
+export interface MessageEvent extends Event {
+    data: any;
+    lastEventId: string;
+}
+
+export class EventSourcePolyfill {
+    static readonly CLOSED: number;
+    static readonly CONNECTING: number;
+    static readonly OPEN: number;
+
+    constructor(url: string, options: EventSourcePolyfillInit);
+
+    onerror: ((this: EventSource, ev: Event) => any) | null;
+    onmessage: ((this: EventSource, ev: MessageEvent) => any) | null;
+    onopen: ((this: EventSource, ev: Event) => any) | null;
+
+    readonly readyState: number;
+    readonly url: string;
+    readonly withCredentials: boolean;
+    close(): void;
+    readonly CLOSED: number;
+    readonly CONNECTING: number;
+    readonly OPEN: number;
```

Comment:
```suggestion
    readonly readyState: number;
    readonly url: string;
    readonly withCredentials: boolean
    readonly CLOSED: number;
    readonly CONNECTING: number;
    readonly OPEN: number;
    
    close(): void;
```

Reply:
yeah, i used the same ordering that's in `lib.dom.d.ts`. accepted suggestion.